addon/routes/-lazy-loader.js

```diff
@@ -10,11 +10,11 @@ export default Ember.Route.extend({
       return;
     }
     retried = !retried;
-    transition.abort();
     var loadPromise = transition.intent.url ?
       this.get('lazyLoader').loadBundleForUrl(transition.intent.url) :
       this.get('lazyLoader').loadBundleForRouteName(transition.intent.name);
     return loadPromise.then(()=>{
+      transition.abort();
       transition.retry();
```

Comment:
Do I understand this right, that you are letting transition proceed, even tho there are no route modules loaded yet. and here aborting transition and then retrying it so it tries to find them again?